workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/ScreenViewFactory.kt

```diff
@@ -177,12 +284,28 @@ public fun interface ViewStarter {
   )
 }
 
+/**
+ * Transforms a [ScreenViewFactory] of [WrappedT] into one that can handle
+ * instances of [WrapperT].
+ *
+ * @param unwrap a function to extract [WrappedT] instances from [WrapperT]s.
+ */
 @WorkflowUiExperimentalApi
 public inline
 fun <reified WrapperT : Screen, WrappedT : Screen> ScreenViewFactory<WrappedT>.unwrapping(
   crossinline unwrap: (wrapperScreen: WrapperT) -> WrappedT,
 ): ScreenViewFactory<WrapperT> = unwrapping(unwrap) { _, ws, e, su -> su(unwrap(ws), e) }
 
+/**
+ * Transforms a [ScreenViewFactory] of [WrappedT] into one that can handle
```

Comment:
- [ ] Should update this to include the use cases from `DecorativeScreenViewFactory`.